File: aroundHubSpringBoot/src/main/java/com/example/aroungHubJavaSpringBoot/aroundHubSpringBoot/data/handler/ProductDataHandler.java
package com.example.aroungHubJavaSpringBoot.aroundHubSpringBoot.data.handler;

import com.example.aroungHubJavaSpringBoot.aroundHubSpringBoot.data.entity.ProductEntity;

public interface ProductDataHandler {

    ProductEntity saveProductEntity(String productId, String productName, int productPrice, int productStock);

    ProductEntity getProductEntity(String productId);
}

File: aroundHubKotlinSpringBoot/src/main/kotlin/com/example/aroungHubKotlinSpringBoot/aroundHubKotlinSpringBoot/data/handler/ProductDataHandler.kt
package com.example.aroungHubKotlinSpringBoot.aroundHubKotlinSpringBoot.data.handler

import com.example.aroungHubKotlinSpringBoot.aroundHubKotlinSpringBoot.data.entity.ProductEntity

interface ProductDataHandler {
    fun saveProductEntity(productId: String, productName: String, productPrice: Int, productStock: Int) : ProductEntity

    fun getProductEntity(productId: String) : ProductEntity
}